requirements.txt

```diff
@@ -0,0 +1,6 @@
+pandas
+boto3
+pyarrow
+fsspec
+s3fs
```

Comment:
FYI when I load `ojo_getters` I get this warning:
```
UserWarning: Your installed version of s3fs is very old and known to cause
severe performance issues, see also https://github.com/dask/dask/issues/10276
```
... Is it fine to ignore this?

Reply:
i didn't get that warning with 3.10.13 python - also v weird given we don't define the version so you'd think you would install the latest release?